packages/firebase_ui_auth/lib/src/screens/profile_screen.dart

```diff
@@ -387,7 +387,7 @@ class _EmailVerificationBadgeState extends State<_EmailVerificationBadge> {
         children: [
           Container(
             decoration: BoxDecoration(
-              color: Colors.yellow,
+              color: Colors.yellow.withOpacity(0.5),
```

Comment:
please make this configurable as mention [here](https://github.com/firebase/FirebaseUI-Flutter/pull/196#issuecomment-1832483440).